lib/screens/lobby.dart

```diff
@@ -20,14 +24,108 @@ class LobbyScreen extends StatelessWidget {
               };
 
               context.read<NavigationService>().showConfirmationDialog(
-                  onConfirmed: signOut,
-                  confirmText: 'Sign Out',
-                  cancelText: 'Return');
+                    onConfirmed: signOut,
+                    confirmText: 'Sign Out',
+                    cancelText: 'Return',
+                  );
             },
             child: Text('log out'),
           )
         ],
       ),
+      body: Container(
+        child: StreamBuilder<List<Game>>(
+            stream: context.watch<DatabaseService>().streamOfCreatedGames,
+            builder: (context, snapshot) {
+              if (snapshot.data == null) {
+                print('still null');
+                return Container();
+              } else {
+                print(' stream builder working');
+                return Padding(
+                  padding: const EdgeInsets.fromLTRB(0, 10, 0, 0),
+                  child: ListView(
+                    children: <Widget>[
+                      for (var game in snapshot.data) GameTile(game)
+                    ],
+                  ),
+                );
+              }
+            }),
+      ),
+    );
+  }
+}
+
+class GameTile extends StatelessWidget {
+  final Game _game;
+  GameTile(this._game);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      key: Key('created_game_tile_${_game.id}'),
+      child: Padding(
+        padding: EdgeInsets.fromLTRB(20, 5, 20, 5),
+        child: Card(
+          elevation: 2,
+          child: Padding(
+            padding: EdgeInsets.all(20),
+            child: Row(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                Padding(
+                  padding: EdgeInsets.fromLTRB(0, 0, 100, 0),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      Text(
+                        '500m away', // will replace with real data in ticket #60
```

Comment:
might be worth making the comment a TODO to make it super easy to find when you're ready